### ISSUE TITLE
Empty page being rendered on Testcafe

### ISSUE THREAD

**Issue by ArvindJoshi-okta** [DevExpress/testcafe-hammerhead#2056]
### What is your Test Scenario?
Navigate to the support page and run some UI tests on it
 
### What is the Current behavior?
Empty page being rendered
 
![image](https://user-images.githubusercontent.com/47287433/59722861-ab564800-91d9-11e9-8c40-b8c43f54ce38.png)

### What is the Expected behavior?

 
Your website URL (or attach your complete example):  https://support.okta.com/help/s/

<details>
<summary>Your complete test code (or attach your test files):</summary>

 
```js
 test('test', async t => {
  await t.navigateTo('https://support.okta.com/help/s');
});
```
</details>
 
### Your Environment details:
 
* testcafe version:                   1.2.1
* node.js version:                    v8.11.1
* command-line arguments:  "testcafe chrome test.js"
* browser name and version: Chrome (any browser would do)
* platform and version:          Chrome 75.0.3770 / Mac OS X 10.14.5

**Comment by LavrovArtem** [comment]
The simple page for reproducing:
```html
<body>
<script>
    var proxy = new Proxy({}, {
        get(target, prop) {
            if (prop === 'document')
                return document;
            else if (prop === 'proxy')
                return proxy;
            else if (prop === 'eval')
                return eval;

            return void 0;
        },

        has(target, prop) {
            return true;
        }
    });

    with(proxy) {
        const { eval, document } = proxy;

        eval('document.body.appendChild(document.createTextNode("Done!"))');
    }
</script>
</body>
```

**Comment by ArvindJoshi-okta** [comment]
@LavrovArtem Any ETA on a potential fix? Thanks!

**Comment by Farfurix** [comment]
@ArvindJoshi-okta 

Hello,

There are several issues we found in your case and we will update this thread as soon as we get any results.

For team:
I found this **partial** duplication of issue https://github.com/DevExpress/testcafe-hammerhead/issues/2003#issuecomment-486204230.

**Comment by shanthkumar043** [comment]
any update on the fix

**Comment by LavrovArtem** [comment]
Currently, we don't have a version with this fix. Please see [TestCafe Release Notes](https://github.com/DevExpress/testcafe/releases) to keep informed.

**Comment by LavrovArtem** [comment]
The fix is published in `testcafe@1.8.6-alpha.1`.
